### PR TITLE
[dagit-bb] Full-width tables

### PR DESCRIPTION
**Pull request by hellendag**
## Summary

Use new full-width table styles throughout Dagit.

image

## Test Plan

View all affected pages in Dagit, verify proper rendering of tables and surrounding content.

### CODE REVIEW

File: js_modules/dagit/packages/core/src/runs/RunsRoot.tsx [hunk 1]
```diff
@@ -101,119 +101,121 @@ export const RunsRoot: React.FC<RouteComponentProps> = () => {
...
-                  Scheduled
-                </TabButton>
-              }
-              id="scheduled"
-            />
-          </Tabs>
-          <Box padding={{bottom: 8}}>
-            <QueryCountdown pollInterval={POLL_INTERVAL} queryResult={queryResult} />
+        <Group direction="column" spacing={8} padding={{horizontal: 24}}>
+          <PageHeader title={<Heading>Runs</Heading>} />
```
Comment (bengotow) [inline js_modules/dagit/packages/core/src/runs/RunsRoot.tsx:107]: This is a bit visually strange (but outside the scope of this PR!)

File: js_modules/dagit/packages/core/src/runs/RunsRoot.tsx [hunk 2]
```diff
@@ -101,119 +101,121 @@ export const RunsRoot: React.FC<RouteComponentProps> = () => {
...
+                  >
+                    Done
+                  </TabButton>
+                }
+                id="done"
+              />
+              <div style={{display: 'flex', alignSelf: 'stretch'}}>
+                <Divider style={{margin: '6px 0px'}} />
+              </div>
+              <Tab
```
Comment (bengotow) [inline js_modules/dagit/packages/core/src/runs/RunsRoot.tsx:184]: If we upgrade this to use our new TabsWIP, I think that the tabs can now have `onClick` and this doesn't need to be a tab with a button inside anymore...

**Review by bengotow — APPROVED** [review]
LGTM! (I didn't pull this down locally though, I trust your changes to the padding + margins in places, etc. are good.)

**Comment by hellendag** [comment]
I'm certain there will be tweaks that need to be made after this, given the change to the horizontal padding. Let's fix them as they appear.

**Pull request merged**